protocol/messenger.go

```diff
@@ -6463,3 +6465,27 @@ func (m *Messenger) handleSyncSocialLinks(message *protobuf.SyncSocialLinks, cal
 func (m *Messenger) GetDeleteForMeMessages() ([]*protobuf.DeleteForMeMessage, error) {
 	return m.persistence.GetDeleteForMeMessages()
 }
+
+func (m *Messenger) ParseSharedURL(url string) (*MessengerResponse, error) {
+	if !strings.HasPrefix(url, baseStatusUrl) {
+		return nil, errors.New("Url should start with '" + baseStatusUrl + "'")
+	}
+
+	group, data, found := strings.Cut(strings.TrimPrefix(url, baseStatusUrl), "/")
+	if !found {
+		return nil, errors.New("Unexpected url format")
+	}
+
+	switch group {
+	case "c":
+		return m.ParseCommuntyShareURL(data, true)
+	case "c#":
```

Comment:
Probably you had some input for this, but I couldn't find what's `c#` in this context?

Reply:
Answered below :)